server/tests/search_providers/belga_360_archive_test.py

```diff
@@ -326,3 +326,33 @@ def get_period(period):
         self.assertEqual(get_period("week"), "20200207")
         self.assertEqual(get_period("month"), "20200114")
         self.assertEqual(get_period("year"), "20190214")
+
+    def test_get_image_renditions(self):
+        item = self.provider.format_list_item(get_belga360_item()[3])
+        guid = "urn:belga.be:360archive:46768825"
+        self.assertEqual(item["_id"], guid)
+        self.assertEqual(item["mimetype"], "application/superdesk.item.picture")
+        self.assertEqual(item["state"], "published")
+        self.assertEqual(item["guid"], guid)
+        self.assertEqual(item["headline"], "FILES - FBL - WC - 2022")
+        self.assertEqual(item["description_text"], "")
```

Comment:
it shouldn't be empty, but rather this one https://github.com/superdesk/superdesk-belga/pull/403/files#diff-d1bf0aaef3bf81b35878ca4902166f4efe8c7697f94834a98ca59cfe4a91ea7bR312